crates/aptos/src/account/fund.rs

```diff
@@ -36,6 +36,30 @@ pub struct FundWithFaucet {
     pub(crate) profile_options: ProfileOptions,
 }
 
+impl FundWithFaucet {
+    /// Create a new instance of this command with the provided amount.
+    /// This is useful for testing.
+    pub fn default_with_amount(amount: u64) -> Self {
+        Self {
+            account: None,
+            amount,
+            faucet_options: FaucetOptions::default(),
+            rest_options: RestOptions::default(),
+            profile_options: ProfileOptions::default(),
+        }
+    }
+
+    pub fn new_for_indexer_testing(amount: u64) -> Self {
+        Self {
+            account: None,
+            amount,
+            faucet_options: FaucetOptions::default(),
+            rest_options: RestOptions::default(),
+            profile_options: ProfileOptions::default(),
+        }
+    }
```

Comment:
This looks the same as `default_with_amount`, do we need both?

Reply:
This seems to be just duplicated. @larry-aptos do you remember why it was added?

Reply:
Good call; you can remove this. this was here to fix the wrong `RestOptions` default, which is fixed below.